x-pack/plugins/observability_solution/profiling/public/views/functions/differential_topn/index.tsx

```diff
@@ -173,7 +173,7 @@ export function DifferentialTopNFunctionsView() {
               baseValue={
                 state.data
                   ? {
-                      totalCount: state.data.TotalCount,
+                      totalCount: state.data.selfCPU,
```

Comment:
why is this camelCased and `TotalCount` Pascal Cased?

Reply:
This was the first version of the implementation. I'm slowly changing it.